app/models/storage.rb

```diff
@@ -407,7 +407,7 @@ def total_unmanaged_vms
   cache_with_timeout(:count_of_vmdk_disk_files, 15.seconds) do
     # doesnt match *-111111.vmdk, *-flat.vmdk, or *-delta.vmdk
     StorageFile.where(:ext_name => 'vmdk').where("base_name !~ '-([0-9]{6}|flat|delta)\\.vmdk$'")
-      .group("storage_id").pluck("storage_id, COUNT(*) AS vm_count")
+      .group("storage_id").pluck(Arel.sql("storage_id, COUNT(*) AS vm_count"))
```

Comment:
Would it make sense to just use `count`?

```suggestion
      .group("storage_id").count
```

```
StorageFile.group(:storage_id).count

   (18.3ms)  SELECT COUNT(*) AS count_all, "storage_files"."storage_id" AS storage_files_storage_id FROM "storage_files" GROUP BY "storage_files"."storage_id"

# => {1=>1596, 3=>21, 4=>42, 2=>140}

Reply:
yeah, I think so.. They're the same thing, right?